### ISSUE TITLE
Multiline comments formatting breaks with 0.44.8

### ISSUE THREAD

**Issue by MaxDesiatov** [nicklockwood/SwiftFormat#619]
I would assume this is either a side-effect of #609 or something that was present before 0.44.8, but currently this snippet:

```swift
public final class TestView {
  /** Initialize a new test view.
  */
  init<V: View>(_ view: V,
                _ subviews: [TestView] = []) {
  }
}
```

is re-formatted as

```swift
public final class TestView {
  /** Initialize a new test view.
      */
  init<V: View>(_ view: V,
                _ subviews: [TestView] = []) {}
}
```

And then on a subsequent formatting iteration as

```swift
public final class TestView {
  /** Initialize a new test view.
          */
  init<V: View>(_ view: V,
                _ subviews: [TestView] = []) {}
}
```

and so on, with the position of `*/` never converging to something fixed.

I know this would be detrimental for performance, but maybe SwiftFormat could do another formatting run internally to verify that it converges to something stable to avoid issues like this in the future? Maybe if not in CLI runs for users, then in the test suite if it's not doing that in the test suite yet?

**Comment by nicklockwood** [comment]
> I know this would be detrimental for performance, but maybe SwiftFormat could do another formatting run internally to verify that it converges to something stable to avoid issues like this in the future? Maybe if not in CLI runs for users, then in the test suite if it's not doing that in the test suite yet?

It does this in the test suite, but unfortunately it only works for cases included in the test suite :-)

I'll add this case, and fix it for the next release.

**Comment by nicklockwood** [comment]
@MaxDesiatov fixed in 0.44.9

**Comment by MaxDesiatov** [comment]
Splendid, thanks!
